test/remote/gateways/remote_fluidpay_test.rb

```diff
@@ -0,0 +1,159 @@
+require 'test_helper'
+
+class RemoteFluidpayTest < Test::Unit::TestCase
+  def setup
+    @gateway = FluidpayGateway.new(fixtures(:fluidpay))
+
+    @amount = 100
+    @credit_card = credit_card('4111111111111111')
+    @declined_card = credit_card('4000000000000002')
+    @options = {
+      order_id: generate_order_id,
+      billing_address: address,
+      description: 'Store Purchase'
+    }
+  end
```

Comment:
Do we need a 3ds test? Lot's of parens up there.. :)

Reply:
I wish we could do a remote test for that! They don't seem to provide a number or other values for testing that. I can check in the unit test though whether the appropriate data gets set on the POST body though!